.github/workflows/daily_ci.yml

```
@@ -77,13 +77,20 @@ jobs:
     outputs:
       groups: ${{ steps.groups.outputs.groups }}
       splits: ${{ steps.groups.outputs.splits }}
+      pythons: ${{ steps.groups.outputs.pythons }}
     steps:
       - id: groups
         run: |
```

actionlint (1.7.12)

[error] shellcheck reported issue in this script: SC2086:info:2:26: Double quote to prevent globbing and word splitting

(shellcheck)